packages/components/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Breaking Change
 
 - `NumberControl` no longer automatically transforms values when rendering `value` into a `<input />` HTML element.
+- `Dashicon` component no longer renders SVGs. If you rely on this component, make sure to load the dashicon font.
```

Comment:
Shouldn't this have gone under `Unreleased`?

Reply:
yep, mistake 😬

Reply:
No worries, I bumped the major version in last week's release 😄